app.rb

```diff
@@ -17,9 +17,7 @@ class Chitter < Sinatra::Base
 end
 
 post '/peep_page' do
-  text = params['text']
-  connection = PG.connect(dbname: 'chitter_database')
```

Comment:
Consider pulling the connection check into a separate file and then it does not need to be repeated for every call to the database such as also in peep.rb